resources/components/Layout/AppLayout.tsx

```diff
@@ -1,7 +1,70 @@
 import React from 'react'
+import Grid from '@mui/material/Grid'
+import SVG from 'react-inlinesvg'
+import {Avatar, Link} from '@mui/material'
+import associationLogo from '../../icons/ParentAssociationLogoWhite.svg'
```

Comment:
maybe rename this to like `DNDLogo` or something, parent association is pretty vague

Reply:
You're right about this! I renamed it to an appropriate name.